src/main/java/fr/sncf/osrd/infra_state/TVDSectionState.java

```diff
@@ -10,17 +12,20 @@
 @SuppressFBWarnings({"URF_UNREAD_PUBLIC_OR_PROTECTED_FIELD"})
 public class TVDSectionState implements DeepComparable<TVDSectionState> {
     public final TVDSection tvdSection;
-    /** Number of train for wich the tvd section is reserved */
+    /** Number of train for which the tvd section is reserved */
     int reservation;
```

Comment:
In which case a tvd section is reserved multiple times?

Reply:
In some cases when using CBTC

Reply:
Indeed, I totally forgot about it.